services/provider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @package       WT JoomShopping content to Joomla Articles
+ * @version       2.0.0
+ * @Author        Sergey Tolkachyov, https://web-tolk.ru
+ * @copyright     Copyright (C) 2023 Sergey Tolkachyov
+ * @license       GNU/GPL http://www.gnu.org/licenses/gpl-3.0.html
+ * @since         1.0.0
+ */
+
+defined('_JEXEC') || die;
+
+use Joomla\CMS\Extension\PluginInterface;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\DI\Container;
+use Joomla\DI\ServiceProviderInterface;
+use Joomla\Event\DispatcherInterface;
+use Joomla\Plugin\Jshopping\Wt_jshopping_content_to_com_content\Extension\Wt_jshopping_content_to_com_content;
+
+return new class implements ServiceProviderInterface {
```

Comment:
В Joomla 4.3+ провайдеры вот так со скобками идут. 
```suggestion
return new class () implements ServiceProviderInterface {
```